base/loading.jl

```diff
@@ -130,7 +130,9 @@ end
 
 const slug_chars = String(['A':'Z'; 'a':'z'; '0':'9'])
 
-function slug(x::UInt32, p::Int)
+slug(x::UInt32, p::Int) = _slug(unsigned(Int)(x), p)
+function _slug(x::T, p::Int) where T<:Unsigned
+    x::T = x
```

Comment:
I don't understand this one --- why isn't it better just to have `x::UInt32`?

Reply:
I was letting it go to what happens "naturally" with `divrem`, but keeping it `UInt32` is surely better.